kalite/coachreports/static/js/coachreports/coach_reports/views.js

```diff
@@ -0,0 +1,343 @@
+var CoachReportView = BaseView.extend({
+
+    template: HB.template('coach_nav/reports-nav'),
+
+    initialize: function(options) {
+
+        this.facility_select_view = new FacilitySelectView({model: this.model});
+        this.group_select_view = new GroupSelectView({model: this.model});
+
+        this.render();
+    },
+
+    render: function() {
+        this.$el.html(this.template());
+        this.$('#group-select-container').append(this.group_select_view.$el);
+        this.$('#facility-select-container').append(this.facility_select_view.$el);
+        this.tabular_view = new TabularReportView({model: this.model});
```

Comment:
Render is intended to be called at pageload, but the TabularReportView instantiation is not ultimately intended to be in the render method.

This should happen as a result of user interaction.